lib/vagrant-service-manager/services/docker.rb

```diff
@@ -0,0 +1,29 @@
+module Vagrant
+  module ServiceManager
+    SUPPORTED_BOXES = ['adb', 'cdk']
+
+    module Service
+      class Docker
+        def initialize(app, env)
+          @app = app
+          @machine = env[:machine]
+          @ui = env[:ui]
+        end
+
+        def call(env)
+          if SUPPORTED_BOXES.include? @machine.guest.capability(:flavor)
```

Comment:
Why is this inside of the docker hook and not at a higher level?  This structure means that every time we add a new service we have to write this same code again.

I am going to LGTM this, but I am opening an issue to get this fixed as we need to get openshift, etc. added.